Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -1,7 +1,98 @@
 import Logging
 
+/// A `LoggerFragment` which implements the default logger message format.
+public func defaultLoggerFragment() -> some LoggerFragment {
+	TimestampFragment().and(LabelFragment().separated(" ")).maxLevel(.trace)
```

Comment:
@0xTim Hm. `.debug` or `.trace` for the timestamps? We're already using `.debug` as the minimum for logging source location...

Reply:
Can we make it configurable? Most logging systems will add their own timestamps but it comes up often enough that it would be nice to turn it on easily

Reply:
If we're leaning towards making it configurable, we may as well just leave the timestamp out from the default one. 

All you have to do to add a timestamp to the default log message now is

```swift
LoggingSystem.bootstrap(
    fragment: TimestampFragment().separated(" ").and(defaultLoggerFragment()),
    console: Terminal()
)
```